env.js

```diff
@@ -1,13 +1,14 @@
 var log = require('loglevel');
 
 module.exports = function() {
-  if (process.env.COUCH2PG_DEBUG) {
+  if (process.env.COUCH2PG_DEBUG || true) {
     log.setDefaultLevel('debug');
   } else {
     log.setDefaultLevel('info');
   }
 
   return {
+    legacyMode: process.env.LEGACY_MODE || false,
```

Comment:
We do have a ticket to put this in gardener, but while we're talking, I'm not really sure why? I mean, tags right? All I'm saying is that every release of couch2pg doesn't have to support every release of medic-webapp. As long as a) we tend toward upgrading people to the latest version of medic-webapp and b) couch2pg is a relatively simple piece of software that doesn't need updates much I don't see the need to get anything more complicated than that.